kubernetes/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: squash-db
+  labels:
+    app: squash-api
+spec:
+  ports:
+    - protocol: TCP
+      port: 3306
+  selector:
+    app: squash-api
+  type: LoadBalancer
```

Comment:
`LoadBalancer` allocates a routable IP and makes the service accessible from anywhere without access control.  This probably a bad thing -- especially considering that containers are not as isolated as VMs.  Things to consider are:

* making this a `ClusterIP` that is not externally accessible.
* Setting up network ACLs along the lines of https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/ (I have not tried this)
* Network policies https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy/ (I have not tried this nor tried to add support for it to a gke cluster)